FreeBSD undefined reference to libiconv

Latest github fails to compile for me on FreeBSD. I ran `git submodule update --init`, and I also have libiconv installed. I'm using GCC, because clang build was giving many errors (I can open a new issue for that if needed)

$ uname -r
11.3-RELEASE-p2

$ pkg info | grep libiconv
libiconv-1.14_11               Character set conversion library

$ gcc --version | head -n 1
gcc (FreeBSD Ports Collection) 9.1.0

$ cc --version ( <----- not used - just including for reference)
FreeBSD clang version 8.0.0 (tags/RELEASE_800/final 356365) (based on LLVM 8.0.0)
Target: x86_64-unknown-freebsd11.3
Thread model: posix
InstalledDir: /usr/bin

[...]
```
gcc    -W -Wall -Wextra -O2 -pipe -std=gnu99 -Iinclude/ -IOpenCL/ -Ideps/LZMA-SDK/C -Ideps/zlib/ -Ideps/OpenCL-Headers -DWITH_BRAIN -Ideps/xxHash src/modules/module_10800.c obj/combined.NATIVE.a -o modules/module_10800.so  -s -lpthread -lm -shared -fPIC -D MODULE_INTERFACE_VERSION_CURRENT=520
/usr/local/bin/ld: obj/combined.NATIVE.a(backend.NATIVE.o): in function `run_cracker':
backend.c:(.text+0x8526): undefined reference to `libiconv'
/usr/local/bin/ld: backend.c:(.text+0x8666): undefined reference to `libiconv_open'
/usr/local/bin/ld: backend.c:(.text+0x8972): undefined reference to `libiconv'
/usr/local/bin/ld: obj/combined.NATIVE.a(wordlist.NATIVE.o): in function `get_next_word.localalias':
wordlist.c:(.text+0x3b1): undefined reference to `libiconv'
/usr/local/bin/ld: obj/combined.NATIVE.a(wordlist.NATIVE.o): in function `count_words':
wordlist.c:(.text+0xa20): undefined reference to `libiconv'
/usr/local/bin/ld: obj/combined.NATIVE.a(wordlist.NATIVE.o): in function `wl_data_init':
wordlist.c:(.text+0xedb): undefined reference to `libiconv_open'
/usr/local/bin/ld: obj/combined.NATIVE.a(wordlist.NATIVE.o): in function `wl_data_destroy':
wordlist.c:(.text+0xf45): undefined reference to `libiconv_close'
/usr/local/bin/ld: obj/combined.NATIVE.a(dispatch.NATIVE.o): in function `calc_stdin':
dispatch.c:(.text+0x25a): undefined reference to `libiconv'
/usr/local/bin/ld: dispatch.c:(.text+0x477): undefined reference to `libiconv_open'
/usr/local/bin/ld: dispatch.c:(.text+0x4a6): undefined reference to `libiconv_close'
collect2: error: ld returned 1 exit status
```

--- a/src/Makefile
+++ b/src/Makefile
@@ -260,6 +260,7 @@ CFLAGS_NATIVE           := $(CFLAGS)
 LFLAGS_NATIVE           := $(LFLAGS)
 LFLAGS_NATIVE           += -lpthread
 LFLAGS_NATIVE           += -lm
+LFLAGS_NATIVE           += -liconv
 endif
 endif # FreeBSD
 

